tests/test_odgovlt.py

```diff
@@ -195,3 +202,13 @@ def test_OdgovltHarvester(app, db, mocker):
     assert result['report_status'] == 'added'
     assert result['errors'] == []
     assert was_last_job_considered_error_free()
+    tag1 = odgovlt.get_package_tags(database_data_list[0]['R_ZODZIAI'])
+    tag2 = odgovlt.get_package_tags(database_data_list[1]['R_ZODZIAI'])
+    assert isinstance(tag1, list)
+    assert isinstance(tag2, list)
+    tag11 = database_data_list[0]['R_ZODZIAI']
+    tag12 = database_data_list[1]['R_ZODZIAI']
+    tag_1 = tag11.lower().replace(u'\u200b', ' ').split(',')
+    tag_2 = tag12.lower().replace(u'\u200b', ' ').split(',')
+    assert tag1 == tag_1
+    assert tag2 == tag_2
```

Comment:
I would suggest to get data from CKAN, using CKAN API and test whole results in single assert statement. This way you will assert everything, types, structure, values, etc...